etl/steps/data/grapher/worldbank_wdi/2025-01-24/wdi.meta.yml

```diff
@@ -1,34 +1,10 @@
-definitions:
```

Comment:
Would it be better to move all this to Garden or are there problems with that?

Reply:
I could try and do that, but there is this unusual message in VS Code and I don't want to ruin the PR:
<img width="433" alt="image" src="https://github.com/user-attachments/assets/05f788a5-e0bd-4ed7-9c54-2451d69a65aa" />

Reply:
Good point! I moved it to `garden/wdi.meta.override.yml` since `wdi.meta.yml` is mostly automatically generated.